### PR TITLE
Player 1897

### PR DESCRIPTION
**Pull request by aeng7**
-added new mute and unmute message bus events
-sync up INITIAL_PLAY jsdoc with mjolnir

### CODE REVIEW

**Comment by pilievOoyala** [comment]
LGTM

File: js/utils/constants.js [hunk 1]
```diff
@@ -585,6 +591,34 @@
...
       /**
+       * A request to change the mute state has been made.
+       * The handler is called with the following arguments:
+       * <ul>
+       *   <li>The desired mute state of the video element.</li>
+       *   <li>The id of the video on which to mute (as defined by the module that controls it).
+       *        If null or undefined, all video elements volume will be changed</li>
+       * </ul>
+       *
+       *
```
Comment (jpfranco) [inline js/utils/constants.js]: Do these need `@private` or `@public` as well?

File: js/utils/constants.js [hunk 2]
```diff
@@ -585,6 +591,34 @@
...
+       * @event OO.EVENTS#CHANGE_MUTE_STATE
+       */
+      CHANGE_MUTE_STATE: 'changeMuteState',
+
+      /**
+       * The mute state has changed.
+       * The handler is called with the following arguments:
+       * <ul>
+       *   <li>The current mute state of the video element.</li>
+       *   <li>The id of the video on which to mute (as defined by the module that controls it).
```
Comment (jpfranco) [inline js/utils/constants.js]: Just a minor nitpick, should this be in past tense to reflect that this already happened?

Comment (aeng7) [reply js/utils/constants.js]: Yep, I'll update this.

File: js/utils/constants.js (hunk 1 above)
Comment (aeng7) [reply js/utils/constants.js]: There is no consistency in the file about that, but I'll add @public since the skin makes use of them.

**Comment by aeng7** [comment]
I'll update the mjolnir constants.js in a separate PR that contains the other changes with the muted events.

**Comment by jpfranco** [comment]
LGTM

**Pull request merged**